bitnami/cilium/templates/agent/daemonset.yaml

```diff
@@ -33,6 +33,9 @@ spec:
         {{- if .Values.agent.cgroup2.autoMount }}
         container.apparmor.security.beta.kubernetes.io/host-mount-cgroup2: "unconfined"
         {{- end }}
```

Comment:
Not related to this but, I see that we don't have a vpa.yaml file here

Reply:
Yes we have it: https://github.com/bitnami/charts/blob/main/bitnami/cilium/templates/agent/vpa.yaml

Reply:
Never mind, it was the vscode UI only showing me the modified files :sweat_smile: